app/assets/stylesheets/earthworks.css

```diff
@@ -120,6 +121,12 @@
   border: 1px solid #ABABA9;
   padding: .5rem;
   border-radius: .25rem;
+  background-color: white;
+}
+
+.navbar-search {
+  --bs-light-rgb: var(--stanford-fog-light-rgb);
+  margin-bottom: 2rem;
```

Comment:
In other projects we've moved this spacing to `#main-container` as you often want spacing from the header even if it's not the `.navbar-search` that's right above it.

Reply:
In fact https://github.com/sul-dlss/earthworks/blob/ff836cc0e8315c2fc81662809588b281ead86aba/app/assets/stylesheets/earthworks.css#L8 is the thing that is removing the default padding from the main container.

Reply:
I think https://github.com/sul-dlss/earthworks/pull/1232 will obviate the need for this margin.

Reply:
Looks like it did. Thanks.